services/core-web/src/constants/NOWConditions.js

```diff
@@ -66,12 +66,12 @@ export const draftPermitMenu = [
     title: "Conditions",
     alwaysVisible: true,
     children: [
-      { href: "GEC", title: "A. General" },
-      { href: "HSC", title: "B. Healthy and Safety" },
-      { href: "GOC", title: "C. Geotechnical" },
-      { href: "ELC", title: "D. Environmental Land" },
-      { href: "RCC", title: "E. Reclamation and Closure" },
-      { href: "ADC", title: "F. Additional Conditions" },
+      { href: "GEC", title: "> General", light: true },
```

Comment:
Why don't we want the lettering anymore. Do we still want these lettered in the generated permit documents?

Reply:
Yes, The numbering is still on the UI in the sections, I think it is still desirable in the permit document. 
This is for the side nav - assuming we removed the lettering because it is left aligned now